react/features/authentication/actions.native.ts

```diff
@@ -65,11 +67,44 @@ export function cancelWaitForOwner() {
     };
 }
 
-/** .
+/**
  * Redirect to the default location (e.g. Welcome page).
  *
  * @returns {Function}
  */
 export function redirectToDefaultLocation() {
     return (dispatch: IStore['dispatch']) => dispatch(appNavigate(undefined));
 }
+
+/**
+ * Opens token auth URL page.
+ *
+ * @param {string} receivedTokenAuthServiceUrl - URL pointing to JWT token authentication service.
+ * @param {string} tokenAuthServiceUrl - Authentication service URL.
+ *
+ * @returns {Function}
+ */
+export function openTokenAuthUrl(receivedTokenAuthServiceUrl: string | undefined, tokenAuthServiceUrl: string) {
```

Comment:
Shouldn't this function always get the URL?

Reply:
Also, what's the deal with the 2 URLs?

Reply:
You are right. I reworked both actions.